akka-stream/src/main/scala/akka/stream/javadsl/Sink.scala

```diff
@@ -85,6 +85,12 @@ object Sink {
   def ignore[T](): Sink[T, CompletionStage[Done]] =
     new Sink(scaladsl.Sink.ignore.toCompletionStage())
 
+  /**
+   * A [[Sink]] that will always backpressure and never consume any elements from the stream.
+   * */
+  def backpressure[T]: Sink[T, CompletionStage[Done]] =
```

Comment:
I think the name `backpressure` may cause some confusion. Looking at corresponding `Source` we have `Source.never`. Could `Sink.never` be a better name?

Reply:
Cool.

Reply:
@patriknw I have updated it with `never`